inyoka_theme_default/static/style/inyoka/portal.less

```diff
@@ -27,12 +27,139 @@
       }
     }
   }
+
   &-usericon {
     font-size:  20ex;
     text-align: center;
     vertical-align: middle;
   }
 
+  .private-message {
+    &-unread {
+      font-weight: bold;
+    }
+  }
+
+  &-privmsg-table {
+    th.actions {
+      text-align: center;
+    }
+    .actions {
+      text-align: right;
+    }
+    th.delete {
+      text-align: center;
+      input {
+        margin-left: 5px;
+      }
+    }
+    .pn_delete {
+      text-align: center;
+    }
+
+    @media (min-width: @screen-md-min) {
+      .actionscol {width:10%;}
+      .deletecol {width:14%;}
+    }
+    @media (min-width: @screen-lg-min) {
+      .actionscol {width:10%;}
+      .deletecol {width:12%;}
+    }
+
+    @media (max-width: @screen-xs-max) {
+      thead {display:none;}
+      tbody {
+        tr {
+          display:block;
+          width:100%;
+          height:auto;
+          border-top:1px solid @gray-lighter;
```

Comment:
„this seems to cause unwanted vertical lines on mobile devices“  from https://github.com/inyokaproject/theme-default/pull/108#discussion_r36793605